app/components/team_icon/team_icon.js

```diff
@@ -49,7 +49,7 @@ export default class TeamIcon extends React.PureComponent {
             teamIconContent = (
                 <Image
                     style={[styles.image, styleImage]}
-                    source={{uri: teamIconUrl}}
+                    source={{uri: teamIconUrl, headers: {Authorization: `Bearer ${Client4.getToken()}`}}}
```

Comment:
if it's breaking why doesn't it need to be on profile pictures too

Reply:
This component is used in the share extension where the headers are not set automatically, but profile pictures are not